README.md

```diff
@@ -8,6 +8,11 @@ at the heart of the [Wikibase software](http://wikiba.se/).
 
 ## Release notes
 
+### 3.1.0 (dev)
+
+* Added `hash` parameters to the `Snak` class constructors
+  and a `getHash()` function to the `Snak` class.
```

Comment:
I think this should mention the actual classes that got a new constructor parameter. For two of them the new parameter is the 2nd, for one it's the 3rd.